client/css/style.css

```diff
@@ -1130,12 +1130,12 @@ kbd {
 	color: #555;
 }
 
-#chat.hide-optional .join,
-#chat.hide-optional .mode,
-#chat.hide-optional .nick,
-#chat.hide-optional .part,
-#chat.hide-optional .quit,
-#chat.hide-optional .condensed,
+#chat.hide-status-messages .join,
+#chat.hide-status-messages .mode,
+#chat.hide-status-messages .nick,
+#chat.hide-status-messages .part,
+#chat.hide-status-messages .quit,
+#chat.hide-status-messages .condensed,
```

Comment:
I merged both options together, and figured this was a more appropriate name than `optional` :man_shrugging: